### PR TITLE
New package: python-ndg_httpsclient-0.4.2

### PR DESCRIPTION
**Pull request by lafrenierejm**
I'm not sure if I'm handling `build_options` correctly here.

### CODE REVIEW

File: srcpkgs/python-ndg-httpsclient/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+wrksrc="ndg_httpsclient-${version}"
+build_style=python-module
+build_options="subjectAltNames"
+build_options_default="subjectAltNames"
+pycompile_module="ndg-httpsclient"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-openssl $(vopt_if subjectAltNames python-pyasn1)"
+short_desc="Pure-Python git object database (Python2)"
+maintainer="Joseph LaFreniere <joseph@lafreniere.xyz>"
+homepage="https://github.com/gitpython-developers/ndg-httpsclient"
```
Comment (Vaelatern) [inline srcpkgs/python-ndg-httpsclient/template]: this page does not resolve.

**Comment by Vaelatern** [comment]
You used build options correctly syntatically, but I'm not sure it makes sense to have a build option in this case.

File: srcpkgs/python-ndg-httpsclient/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+distfiles="${PYPI_SITE}/n/ndg_httpsclient/ndg_httpsclient-${version}.tar.gz"
+checksum=580987ef194334c50389e0d7de885fccf15605c13c6eecaabd8d6c43768eb8ac
+
+python3-ndg-httpsclient_package() {
+	noarch=yes
+	depends="python3-openssl $(vopt_if subjectAltNames python-pyasn1)"
+	pycompile_module="ndg-httpsclient"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove usr/lib/python3*
```
Comment (asergi) [inline srcpkgs/python-ndg-httpsclient/template]: `vlicense` here too.

File: srcpkgs/python-ndg-httpsclient/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+license="BSD"
+distfiles="${PYPI_SITE}/n/ndg_httpsclient/ndg_httpsclient-${version}.tar.gz"
+checksum=580987ef194334c50389e0d7de885fccf15605c13c6eecaabd8d6c43768eb8ac
+
+python3-ndg-httpsclient_package() {
+	noarch=yes
+	depends="python3-openssl $(vopt_if subjectAltNames python-pyasn1)"
+	pycompile_module="ndg-httpsclient"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
```
Comment (asergi) [inline srcpkgs/python-ndg-httpsclient/template]: `vmove usr/bin/*3` and set `alternatives`.

File: srcpkgs/python-ndg-httpsclient/template [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+maintainer="Joseph LaFreniere <joseph@lafreniere.xyz>"
+homepage="https://github.com/cedadev/ndg_httpsclient"
+license="BSD"
+distfiles="${PYPI_SITE}/n/ndg_httpsclient/ndg_httpsclient-${version}.tar.gz"
+checksum=580987ef194334c50389e0d7de885fccf15605c13c6eecaabd8d6c43768eb8ac
+
+python3-ndg-httpsclient_package() {
+	noarch=yes
+	depends="python3-openssl $(vopt_if subjectAltNames python-pyasn1)"
+	pycompile_module="ndg-httpsclient"
```
Comment (asergi) [inline srcpkgs/python-ndg-httpsclient/template]: Ditto.

File: srcpkgs/python-ndg-httpsclient/template [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'python-ndg-httpsclient'
+pkgname=python-ndg-httpsclient
+version=0.4.2
+revision=1
+noarch=yes
+wrksrc="ndg_httpsclient-${version}"
+build_style=python-module
+build_options="subjectAltNames"
+build_options_default="subjectAltNames"
```
Comment (asergi) [inline srcpkgs/python-ndg-httpsclient/template]: Remove `build_options` et al. and keep `pyasn1` to `depends`.
Anyway: `python-openssl` -> `python-cryptography` -> `python-pyasn1`.

**Pull request merged**